### ISSUE TITLE
Unable to open .NIF file (failed to load block number)

### ISSUE THREAD

**Issue by jacksw32** [hexabits/nifskope#64]
I cannot open this file for some reason, it gives me the error:  failed to load block number 16 (NiDataStream) previous block was NiMaterialProperty,  I am currently using Nifskopes latest version and for debugging purposes I attached the file here too. 
[Moskvoretsky_bridge.zip](https://github.com/hexabits/nifskope/files/12807861/Moskvoretsky_bridge.zip)

**Comment by hexabits** [comment]
There may have been a regression in nifxml,  I'll take a look.

**Comment by jacksw32** [comment]
> There may have been a regression in nifxml, I'll take a look.

Alright is there a fix for it?

**Comment by hexabits** [comment]
No, not yet.  The latest nifxml had some changes to the specification that I didn't know about and it affects NiDataStream reading.  I will have to find a way to compensate for it. 

You can however revert that part of nif.xml if you want.  Change:

```xml
<field name="Data" type="DataStreamData" arg1="Num Bytes" arg2="Component Formats" />
```
to
```xml
<field name="Data" type="byte" binary="true" length="Num Bytes" />
```

Should be around Line 7200.

**Comment by jacksw32** [comment]
> No, not yet. The latest nifxml had some changes to the specification that I didn't know about and it affects NiDataStream reading. I will have to find a way to compensate for it.
> 
> You can however revert that part of nif.xml if you want. Change:
> 
> ```
> <field name="Data" type="DataStreamData" arg1="Num Bytes" arg2="Component Formats" />
> ```
> 
> to
> 
> ```
> <field name="Data" type="byte" binary="true" length="Num Bytes" />
> ```
> 
> Should be around Line 7200.

Alright thank you. I will try it and see if it works :)
